pyproject.toml

```diff
@@ -18,6 +20,12 @@ dev = [
   "typing-extensions",
 ]
 
+[build-system]
+requires = ["setuptools >= 77.0.3", "setuptools-scm>=8", "wheel"]
```

Comment:
I was going to use `uv_build` instead of `setuptools`. I haven't compared `uv_build` and `setuptools` performances, and `uv_build` doesn't support dynamic metadata yet, so I guess `setuptools` is fine :-)